Week-3/exercise_3_1/index.js

```diff
@@ -0,0 +1,26 @@
+//Given reducer method:
+const add = (a, b = 0) => a + b;
```

Comment:
@Saurabhrkp the assignment asks for n number of args. This fn only processes 2

Reply:
Okay, making add function to get n number of args

Reply:
done